### DISCUSSION TITLE
Broken SKU problem in Etsy

### DISCUSSION THREAD

**Discussion by jakeleventhal** [etsy/open-api#1332] [Ideas]
In Etsy, whether you do it via the API or via the Etsy dashboard, if you have some variants with SKUs and others with empty SKUs, Etsy will just randomly assign one of the other variant's SKUs to the ones that were set to empty.

**Comment by Norithiel** [comment]
Same for me!

**Comment by SrdjanPaunovic** [comment]
I had the same problem. My workaround was to assign a placeholder value for the SKU (e.g., 'Empty' or a dummy value), which I later ignored. However, it would be great to have this issue fixed, as assigning random values is far from ideal and could lead to customers receiving incorrect items.
